pkgs/tools/X11/xidlehook/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, rustPlatform, fetchFromGitHub, x11, xorg, libpulseaudio, pkgconfig, makeWrapper }:
+
+rustPlatform.buildRustPackage rec {
+  name = "xidlehook-${version}";
+  version = "0.4.5";
+
+  src = fetchFromGitHub {
+    owner = "jD91mZM2";
+    repo = "xidlehook";
+    rev = version;
+
+    sha256 = "0mj02w959h0sr8b2r3l54dqn4j3y88xdr434y3jyfqd371kg0i39";
```

Comment:
I am getting another hash here:
```
unpacking source archive /build/0.4.5.tar.gz
fixed-output derivation produced path '/nix/store/f78zppjisrw0chk8dmx3vifqrwzkshql-source' with sha256 hash '1jdy85sh7w8drcahx33swqwnv2ik7rpxl0hc07waxw3iwwk63m66' instead of the expected hash '0mj02w959h0sr8b2r3l54dqn4j3y88xdr434y3jyfqd371kg0i39'
```

Reply:
Whoops, I forgot to update the hash since 0.0.4. My bad!